hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/dstag/DistributionTagButtonClick.java

```diff
@@ -18,13 +19,16 @@
 import com.vaadin.ui.Button;
 
 /**
- *
- *
+ * Abstract class for button click behavior of the distribution set's tag
+ * buttons. Filters the distribution sets according to the active tags. Is used
+ * on the Deployment View.
```

Comment:
This will be outdated very quickly. Look at the complete chain:
DistributionTagButtonClick  => DistributionTagButtons =>  DistributionTagLayout => DeploymentView

Nobody will adapt this comment if there is a change in one of the dependencies.